### PR TITLE
Integrate sessions API on My sessions route

### LINKED ISSUE #694: My sessions pages shows random sessions without any connection to the user
It seems the "My sessions page" is not implemented as related to the actual user. It currently shows private information of others.

Wrong:
screenshot from 2017-08-13 16-53-22

Previous Implementation is actually working:
screenshot from 2017-08-13 16-55-06

### PR DESCRIPTION
**Pull request by CosmicCoder96**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Integrate sessions API on My sessions route

#### Changes proposed in this pull request:

- Split my sessions int dynamic routes
- Integrate sessions API

Fixes #694

### CODE REVIEW

**Comment by niranjan94** [comment]
Deployment link: https://pr-724-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-724-open-event-frontend-fossasia.surge.sh

File: app/routes/my-sessions/list.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      ];
+    } else if (params.session_status === 'past') {
+      filterOptions = [
+        {
+          name : 'ends-at',
+          op   : 'lt',
+          val  : moment().toISOString()
+        }
+      ];
+    } else {
```
Comment (geekyd) [inline app/routes/my-sessions/list.js]: This else clause is not required we only have two `session_status` i.e past & upcoming

**Comment by geekyd** [comment]
@niranjan94 @CosmicCoder96 Should the sessions be sorted in descending order of event date?

**Comment by CosmicCoder96** [comment]
@geekyd removed the extra else and sorted them. Please review

**Comment by niranjan94** [comment]
Deployment link: https://pr-724-open-event-frontend-fossasia.surge.sh

**Pull request merged**